README.md

```diff
@@ -52,6 +52,14 @@ Similar tools like [KubeDB](https://kubedb.com/docs/0.9.0/concepts/databases/mon
 
 Right now this project is pre alpha and **not recommended for production systems.**
 
+## Tests
+
+This project is supported by unit and integration level tests.
+
+To run the unit-tests you need to provide a mongod instance without auth at localhost and run `npm run test-dev`.
```

Comment:
Do we not want to remove the `mongod` instance from our unit tests?

Reply:
You mean move the tests of that to the integration tests?

Reply:
Yes, I think any test that requires a mono setup should b semi integration no? 

Also, I  feel we are losing out by only testing those against Mongo:4.0, I would like to verify the commands  return the same values on each Mongo version too.

Reply:
I agree.

This also creates a clear seperation between unit and integraiton tests. 

Integration tests use a real mongo instnace. Unit tests are 100% stubbed